ui/collapsible-panel.reel/collapsible-panel.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @module ui/collapsible-panel.reel
+ * @requires montage/ui/component
+ */
+var Component = require("../component").Component,
+    PressComposer = require("../../composer/press-composer").PressComposer;
+
+/**
+ * @class CollapsiblePanel
+ * @extends Component
+ *
+ * A collapsible panel that can be expanded or collapsed with header and body
+ */
+exports.CollapsiblePanel = Component.specialize(/** @lends CollapsiblePanel# */ {
+    constructor: {
+        value: function CollapsiblePanel() {
+            this.super();
+            this._pressComposer = new PressComposer();
+        }
+    },
+
+    _pressComposer: {
+        value: null
+    },
+
+    /**
+     * Set this to true when clicking on header to collapse or expand a panel, only trgiiger animation at this stage.
+     * No animation when first time drawing
+     * @private
+     */
+    _isCollapsing: {
+        value: false
+    },
+
+    _isExpanded: {
+        value: null
+    },
```

Comment:
I see the _xxx property is defined in some other montage files and some not, like I see in select.js, the _content and _selectedIndexes and _contentController are all defined. I think it's good to always define the private property for setter and getter? for me it's more clear. So what do you think? we keep it or remove it?